src/sidebar/components/test/tag-editor-test.js

```diff
@@ -6,6 +6,7 @@ import TagEditor from '../tag-editor';
 import { $imports } from '../tag-editor';
 
 import { checkAccessibility } from './accessibility';
+import AutocompleteList from '../autocomplete-list';
```

Comment:
Can you group this with the other imports from the `src/sidebar/components`, rather than `src/sidebar/components/test` directory above? Imports are grouped in three categories: 1) Vendor/third-party imports, 2) Imports from other directories than the current module, 3) Imports from the same directory as the current module.

nb. It is still on my TODO list to automate this.

Reply:
@robertknight oh I missed that one.

In general it looks like we have 3 groups. Is my understanding correct?

1. boiler plate imports from third party libs

2. imports from files we need to test against

3. imports from test utils


Is this correct?